src/models/dst-container.ts

```diff
@@ -0,0 +1,129 @@
+import { types } from "mobx-state-tree";
+import { DstDataDisplayModel } from "./data-display-model";
+import { DataSet } from "../codap/models/data/data-set";
+import { SharedCaseMetadata } from "../codap/models/shared/shared-case-metadata";
+import { IHistoryServiceEnv } from "../codap/models/history/history-service";
+
+export const DstContainer = types.model("DstContainer", {
+  dataDisplayModel: DstDataDisplayModel,
+  dataSet: DataSet,
+  sharedCaseMetadata: SharedCaseMetadata,
+});
+
+const historyServiceEnv: IHistoryServiceEnv = {
+  historyService: {
+    handleApplyModelChange(options) {
+      // noop
+    },
+    withoutUndo(actionCall, options) {
+      // noop
+    }
+  }
+};
+
+export const dstContainer = DstContainer.create({
```

Comment:
Are the details of this object important? For example, do the ids have to match with the dataset's or something?

Reply:
Sort of. I'm adding a comment about how this is initial model is used.